README.md

```diff
@@ -38,6 +38,11 @@ Configuration options for fluent.conf are:
 * `compress_encoding` - Compression encoding format, either `gzip` or `deflate` (default `gzip`)
 * `custom_fields` - Comma-separated key=value list of fields to apply to every log. [more information](https://help.sumologic.com/Manage/Fields#http-source-fields)
 * `custom_dimensions` - Comma-separated key=value list of dimensions to apply to every metric. [more information](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source/Upload-Metrics-to-an-HTTP-Source#supported-http-headers)
+* `retry_forever` - If yes, plugin will be sending data until it sends it successfully
```

Comment:
I think I will remove it, as `retry_forever` is equal to `retry_timeout`=`0` and `retry_max_times`=0